ifj2017/tests/02_functions/tests.json

```diff
@@ -14,13 +14,13 @@
     },
     {
       "info": "Asc value",
-      "stdout": " 104",
-      "code": "scope \n print asc(!\"ahoj\", 2); \n end scope"
+      "stdout": " 111",
+      "code": "scope \n dim a as integer = asc(!\"ahoj\", 2) \n print a; \n end scope"
```

Comment:
Není v základním zadání volání funkce povoleno pouze jako id = funkce(seznam_parametrů) ?

Ano moje zadání neodpovídalo ale řekl bych že tvé také ne. Mohu se mýlit.

Reply:
Souhlas s @martinkobelka, musí tam být samostatné přiřazení.

Reply:
Áno to je pravda. Opravené.